workshops/templates/workshops/training_progresses_inline.html

```diff
@@ -12,6 +12,6 @@
   </a>
   &nbsp;
 {% endfor %}
-<a href="{% url 'trainingprogress_add' person.id %}?next={{ request.get_full_path|urlencode }}"
+<a href="{% url 'trainingprogress_add' %}?trainee={{ person.id }}&next={{ request.get_full_path|urlencode }}"
```

Comment:
I discovered [django-spurl](https://github.com/j4mie/django-spurl). It allows manipulating urls in templates.

```
{% url 'trainingprogress_add' as trainingprogress_add_url %}
<a href="{% spur base=trainingprogress_add_url add_query="trainee={{ person.id }}&next={{ request.get_full_path|urlencode }}" }">
```

I don't think it makes it any easier. We should probably stick to doing it the pure Django way.